data/rulesets.ts

```diff
@@ -1585,6 +1585,12 @@ export const Rulesets: {[k: string]: FormatData} = {
 		desc: "Prevents Pok\u00e9mon from having moves that would only be obtainable in Pok\u00e9mon Crystal.",
 		// Implemented in mods/gen2/rulesets.ts
 	},
+	agilitypartialtrappingclause: {
+		effectType: 'ValidatorRule',
+		name: 'Agility + Partial Trapping Clause',
```

Comment:
I don't like the + being in the name here, can it be renamed to AgiliTrap or something?

Reply:
That's not really a "used" term. If you want something else, APT would be the appropriate name, which was used in the suspect test's terminology. 

https://www.smogon.com/forums/threads/rby-uu-agility-partial-trapping-apt-suspect-test-apt-banned.3700597/

Reply:
I don't really care what the name is, as long as there isn't a + in the rule name. It doesn't functionally break anything, but the ambiguity with how unbans work in challenges/how complex bans work in challenges makes me prefer that this was just manually added to the UU ruleset directly. There also doesn't need to be a clause banning these if only one format bans them in the first place, so I'm unsure if this is even a necessity.

Reply:
Aye, right. We can go with APT, but lemme just verify this last bit before I change stuff, just to be sure.

> There also doesn't need to be a clause banning these if only one format bans them in the first place, so I'm unsure if this is even a necessity.

I'm mainly going in with the rationale that it's highly likely other RBY tiers will ban the combination, NU especially. If it's not a concern (we _could_ "cross that bridge when we come to it", I try to make things easier later) then yeah we can just do the ban that way. Let me know what you think.

Reply:
I do not like Chris "Cross That Bridge" Christie so lets just do APT